### ISSUE TITLE
Unable to find some Wifi Networks

### ISSUE THREAD

**Issue by rubencarneiro** [ubports/ubuntu-touch#1150]
- Turbo
- Stable/Dev
- Build:2019-06-03

### Steps to reproduce
Try to find some wifi network with android installed.

### Expected behavior
Find Network without any problem

### Actual behavior
With ubuntu some networks that appears on android, dont appear on ubuntu, and im next to the wifi router sometimes.

**Comment by MK1973** [comment]
Yes, i can confirm this issue and add that sometimes si see the WiFi network but most of the time not. 

It happens to me always with the same network. And it is since long time. 

N5, dev, RC, stable OTA9

**Comment by kugiigi** [comment]
Can you confirm that the android device is exactly the same with UT?
Perhaps the Wifi frequency is not really supported by your UT device?
For example, bq E5 does not support 5Ghz Wifi.

**Comment by rubencarneiro** [comment]
Yes i tested the exactly device with android and Ubuntu touch.
For this only happens on Meizu Pro 5.

**Comment by Flohack74** [comment]
Network Manager update is being planned but will still take some time. Hopefully it will help. This is definitely not a problem of the Android part, unfortunately.

**Comment by rubencarneiro** [comment]
This is related to ubuntu specific.

**Comment by UniversalSuperBox** [comment]
For anyone who was having trouble with this on the Nexus 5, please head over to https://github.com/ubports/ubuntu-touch/issues/714. I believe these are separate issues.
